lib/cmake/mxl-config.cmake.in

```diff
@@ -1,5 +1,5 @@
 @PACKAGE_INIT@
 
 if(NOT TARGET mxl::mxl)
-    include("${CMAKE_CURRENT_LIST_DIR}/mxlTarget.cmake")
+    include("${CMAKE_CURRENT_LIST_DIR}/mxl-targets.cmake")
```

Comment:
Looks correct to me. This needs to match the filename produced by the `install(EXPORT)` command. The default filename if `FILE` isn't explicitly specified is as given here.

* https://github.com/dmf-mxl/mxl/blob/4ca63a206f9e308ad10fdb27acea2ee906df278a/lib/CMakeLists.txt#L158-L162
* https://cmake.org/cmake/help/latest/command/install.html#export